scripts/slither.ts

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+
+import fs from "fs";
+import inquirer from "inquirer";
+import { execSync } from "node:child_process";
+import path from "node:path";
+
+const projectRoot = path.join(__dirname, "../");
+const solcVersion = "0.8.12";
+const timestamp = Date.now();
+
+async function getPackageName() {
+  const packageName = await inquirer.prompt([
+    {
+      type: "list",
+      message: "Which set of contracts would you like to test?",
+      name: "contracts",
+      choices: ["protocol-contracts", "example-contracts"],
+    },
+  ]);
+  return packageName.contracts;
+}
+
+const run = async (command: string) => {
+  try {
+    console.log("Starting -- This may take a few minutes...");
+    execSync(command, {
+      encoding: "utf-8",
+      stdio: "inherit",
+    });
+    console.log("Results output to the console and saved to slither-output/ in Markdown, JSON, and SARIF formats.");
+  } catch (error: any) {
+    console.error("Error: Docker Failed To Run");
+    console.error(`${error}`);
+  }
+};
+
+function runSlither(packageName: string) {
+  const dockerCommand = `cd /home/trufflecon/packages/${packageName} && \
+  solc-select use ${solcVersion} && \
+  slither --json ../../scripts/slither-results/${packageName}-${timestamp}.json \
+  --sarif ../../scripts/slither-results/${packageName}-${timestamp}.sarif \
+  --checklist  ./ | tee ../../scripts/slither-results/${packageName}-${timestamp}.md`;
+  run(`docker run -v "${projectRoot}":/home/trufflecon trailofbits/eth-security-toolbox  -c "${dockerCommand}"`);
+}
+
+function createOutputDir() {
+  var dir = "./slither-results";
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir);
+  }
+}
+
```

Comment:
Let's delete this and create a `slither-results` folder with a `.gitkeep` file inside